examples/Buttons/Buttons.ino

```diff
@@ -0,0 +1,106 @@
+/*
+Buttons example
+June 11, 2015
+Copyright (C) 2015 David Martinez
+All rights reserved.
+This code is the most basic barebones code for showing how to use buttons in
+Arduboy.
+
+This library is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 2.1 of the License, or (at your option) any later version.
+*/
+
+#include "Arduboy.h"
+
+// Make an instance of arduboy used for many functions
+Arduboy arduboy;
+
+// Variables for your game go here.
+char text[] = "Press Buttons!";
+byte x;
+byte y;
+
+// Width of each charcter including inter-character space
+#define CHAR_WIDTH 6
+
+// Height of each charater
+#define CHAR_HEIGHT 8
+
+// To get the number of characters, we subtract 1 from the length of
+// the array because there will be a NULL terminator at the end.
+#define NUM_CHARS (sizeof(text) - 1)
+
+// This is the highest value that x can be without the end of the text
+// going farther than the right side of the screen. We add one because
+// there will be a 1 pixel space at the end of the last character.
+// WIDTH and HEIGHT are defined in the Arduboy library.
+#define X_MAX (WIDTH - (NUM_CHARS * CHAR_WIDTH) + 1)
+
+// This is the highest value that y can be without the text going below
+// the bottom of the screen.
+#define Y_MAX (HEIGHT - CHAR_HEIGHT)
+
+
+// This function runs once in your game.
+// use it for anything that needs to be set only once in your game.
+void setup() {
+  //initiate arduboy instance
+  arduboy.begin();
+
+  // here we set the framerate to 30, we do not need to run at default 60 and
+  // it saves us battery life.
+  arduboy.setFrameRate(30);
+  
+  // set x and y to the middle of the screen
+  x = (WIDTH / 2) - (NUM_CHARS * CHAR_WIDTH / 2);
+  y = (HEIGHT / 2) - (CHAR_HEIGHT / 2);
+}
+
+
+// our main game loop, this runs once every cycle/frame.
+// this is where our game logic goes.
+void loop() {
```

Comment:
Could we stick to the style of the library here and do newlines before the curly-brace bracket?

Example:

```
void loop()
{
  ...
}
```

This would promote the same style throughout the repository, and it does make the function separation easier to read.